modules/govuk_jenkins/templates/jobs/deploy_emergency_banner.yaml.erb

```diff
@@ -25,7 +25,59 @@
           LINK=$(escapeRakeArg $LINK)
```

Comment:
I wonder if you want to add `set -ex` to this file so it can output everything and break on errors.

Reply:
What does this do?

Reply:
-e will make it stop executing if one of the commands returns a non 0 exit code
-x will echo out the commands that are being run (which should make it easier to follow progress)

Reply:
Sounds good.